### ISSUE TITLE
MSCOLAB : synchronize takeoff time

### ISSUE THREAD

**Issue by jgrooss** [Open-MSS/MSS#1852]
It seems that the displayed times depend on the performance settings but also on the takeoff time set in the performace dialogue. While it wouldbe difficult for the performace settings, it would be desirable that also the takeoff time is communicated to the MSCOLAB server and back to the others.

**Comment by ReimarBauer** [comment]
Does it make sense to have all performance data synced to all participants of an operation by mscolab?

**Comment by jgrooss** [comment]
Yes that would be even nicer, but they should use the same performance settings anyway. Takeoff date and time are the information that would change from flight plan to flight plan

**Comment by ReimarBauer** [comment]
@jgrooss can you please picture how the interface should be for setting that takeoff time?  

We could maybe set that only Operation Admins and Creator have the right to change it, and it gets synced to all particpants of the operation.

**Comment by jgrooss** [comment]
Takeoff time is normally set in Table view -> Performance Settings -> takeoff time 
There is also the setting of aircraft weight (with/without fuel) that is needed for the estimate of ceiling altitude.
The easiest logic would be that the creator of the MSCOLAB operation also transfers these three numbers to the MSCOLAB server, an each person joining this mscolab op will get these numbers synchronized automatically.
Rights to change theses numbers would be the same as rights to change waypoints.

**Comment by ReimarBauer** [comment]
@jgrooss I think we can add the "estimated Takeoff time" to the operation model definition and a value have to be entered as initial value on creation of an operation.

When the value is changed it gets stored in the operation.takeoff_time (is there a shortcut for that value?)
We can then use the events when it gets changed in the table to sync it to all others. And when someone later accesses the OP the value is fetched from the database.
